integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -38,7 +38,7 @@
     "ProjectExec",
     "SerializeFromObjectExec",
     "SortExec"
-]
+] + ["DataWritingCommandExec", "DeltaOptimizedWriterExec"] if is_databricks122_or_later() else []
```

Comment:
DeltaOptmizedWriterExec sounds like something we should be replacing...?

Reply:
DataWritingCommandExec also sounds like something we should be replacing, if we're supposed to be replacing the write during the test.

Reply:
I have reverted these changes and skipped the failing tests, linking to the follow-on issue https://github.com/NVIDIA/spark-rapids/issues/8423